<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>221</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#333333;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>91</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;
font-size: 15px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#71A031;
font-size:15px;
</string>
    </property>
    <property name="text">
     <string>RaSk0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Choose your Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>3</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
background-color: rgba(255, 255, 255, 0%);
color:#726F6F;
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>1</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
background-color: rgba(255, 255, 255, 0%);
color:#726F6F;
</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>193</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Status: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>191</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>164</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Always on top</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_information">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>7</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0%);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>36</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="styleSheet">
        <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;
 </string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_edit">
       <property name="styleSheet">
        <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="styleSheet">
        <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>131</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_copy_username">
       <property name="styleSheet">
        <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
       </property>
       <property name="text">
        <string>Username/Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_copy_passsword">
       <property name="styleSheet">
        <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_setting">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>6</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/resources/setting-gear.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_timer">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>165</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
</string>
    </property>
    <property name="text">
     <string>Timer: &lt;font color=red&gt;OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4_exit_timer">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>165</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>AutoExit: &lt;font color=red&gt;OFF</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
